assets/src/infrastructure/ui/layout/textfit/TextFit.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import shallowEqual from './utils/shallowEqual';
+import series from './utils/series';
+import whilst from './utils/whilst';
+import throttle from './utils/throttle';
+import uniqueId from './utils/uniqueId';
+import { innerWidth, innerHeight } from './utils/innerSize';
+
+function assertElementFitsWidth(el, width) {
+	// -1: temporary bugfix, will be refactored soon
+	return el.scrollWidth - 1 <= width;
+}
+
+function assertElementFitsHeight(el, height) {
+	// -1: temporary bugfix, will be refactored soon
+	return el.scrollHeight - 1 <= height;
+}
+
+function noop() {}
+
+export default class TextFit extends React.Component {
```

Comment:
In another PR we should convert this to a functional component.

Reply:
sure, will create an issue, to have this tracked

Reply:
https://github.com/eventespresso/event-espresso-core/projects/6#card-38428952

Reply:
thnx guys